search-app/src/cmr/search/services/parameters/converters/shapefile.clj

```diff
@@ -40,6 +40,15 @@
   "Flag that indicates if we allow spatial searching by shapefile."
   {:default false :type Boolean})
 
+
+(defconfig max-shapefile-features
+  "The maximum number of feature a shapefile can have"
+  {:default 100000 :type Long})
+  
+(defconfig max-shapefile-points
+  "The maximum number of points a shapefile can have"
+  {:default 5000000 :type Long})
```

Comment:
I believe this should be less than the max shapefile size. Even if i assume 1 byte per point thats 5mb where `max-shapefile-size` is 1mb. Or am I not accounting for some sort of compression?

Reply:
Yeah - these were wrong. The max number of points should be 100,000 and the max number of features 5000.  These were arrived at through some simple testing with shapefiles of various size/complexity. There are additionally performance tests scheduled for down the road. These values are intended to be intentionally low for now until performance tests can establish higher safe numbers.